admin/git.md

```diff
@@ -23,7 +23,7 @@ Instead, you can use Visual Studio Code with git, the command-line tool, and we
 expect that this combination will work with most hosting software or services.
 However, Coder doesn't test these and cannot provide recommendations or support.
 
-**Note** The public key provided by Coder uses the ECDSA key algorithm. Ensure
+**Note** The public key provided by Coder uses the ED25519 key algorithm. Ensure
```

Comment:
I'm not an expert at this kind of thing, but [according to the Wikipedia page](https://en.wikipedia.org/wiki/EdDSA#Ed25519), it should be stylized "Ed25519"

```suggestion
**Note** The public key provided by Coder uses the Ed25519 key algorithm. Ensure
```